### PR TITLE
Katie deployment debugging

### PR DESCRIPTION
**Pull request by ktracy2**
Deployment is live on DO, but CSS/other static files are still inaccessible.  This branch is a mess and I apologize :(

### CODE REVIEW

File: src/nestdoor/settings.py [hunk 1]
```diff
@@ -30,10 +28,12 @@
 
 # SECURITY WARNING: don't run with debug turned on in production!
 # Edited for Digital Ocean Deployment
-DEBUG = os.getenv("DEBUG", "False") == "True"
+DEBUG = os.getenv("DEBUG", "True") == "True"
 
 # Edited for Digital Ocean Deployment
-ALLOWED_HOSTS = os.getenv("DJANGO_ALLOWED_HOSTS", "nestdoor-app-zcquw.ondigitalocean.app", "127.0.0.1,localhost").split(",")
```
Comment (janunine) [inline src/nestdoor/settings.py]: Pls review this line.

File: src/nestdoorapp/urls.py [hunk 2]
```diff
@@ -5,13 +5,12 @@
 from django.conf.urls.static import static
 
 urlpatterns = [
-    path('admin/', admin.site.urls),
```
Comment (janunine) [inline src/nestdoorapp/urls.py:8]: Pls review this line

**Review by janunine — CHANGES_REQUESTED** [review]

File: src/nestdoor/settings.py (hunk 1 above)
Comment (ktracy2) [reply src/nestdoor/settings.py]: What do you mean by review this line?  Are you referring to the ALLOWED_HOSTS?

File: src/nestdoorapp/urls.py (hunk 2 above)
Comment (ktracy2) [reply src/nestdoorapp/urls.py:8]: I've added it back in my source code which has been pushed to the branch

**Review by janunine — APPROVED** [review]

**Pull request merged**